Tavisca.Bootcamp.LanguageBasics.Exercise2/Program.cs

```diff
@@ -13,18 +13,96 @@ static void Main(string[] args)
+        {
+            //
+            //checking for any conflicts in two showposttime
+            //
+           for (int i = 0; i < exactPostTime.Length; i++)
+            {
+                for (int j = i + 1; j < exactPostTime.Length; j++)
+                 {
+                    if (exactPostTime[i] == exactPostTime[j])
+                        if (showPostTime[i] != showPostTime[j])
+                            return "impossible";
+                }
+            }
+            //
+            //now this block will be executed if there is no conflict in showposttime
+            //
+            string[] myStrings = new string[exactPostTime.Length];
+            for(int i = 0; i<exactPostTime.Length; i++)
+             {
+                string[] my=exactPostTime[i].Split(':');
+                //
+                //for showposttime containing seconds
+                //
+                if(showPostTime[i].Contains("seconds"))
+                {
+                    myStrings[i]=exactPostTime[i];
+                    
+                }
+                //
+                //for showposttime containing minutes
+                //
+                if(showPostTime[i].Contains("minutes"))
+                {
+                    int r=Convert.ToInt32(showPostTime[i].Split(" ")[0]);
+                    int q=Convert.ToInt32(my[1]);
+                    int y=Convert.ToInt32(my[0]);
+                    r=r+q;//calculating total minutes
+                    //if minutes>59
+                    if(r>59)
+                    {
+                        q=r%60;
+                        y=y+r/60;
+                    }
+                    else{
+                        q=r;
+                    }
+                
+                    string t="";
+                    //if after addition hour >23
+                    if(y>23){
+                        t="00";
+                    }
+                    else{
+                        t=Convert.ToString(y);
+                    }
+                    string z=t+":"+Convert.ToString(q)+":"+my[2];
+                    
+                    myStrings[i]=z;
+                    
+                }
+                //
+                //for showposttime containing hours
+                //
+                if(showPostTime[i].Contains("hours"))
+                {
+                    int r=Convert.ToInt32(showPostTime[i].Split(" ")[0]);
+                    int y=Convert.ToInt32(my[0]);
+                    r=r+y;
+                    //if hours>23 after addition
+                    if(r>23)
+                    {
+                        r=r-24;
+                    }
+                    string z=Convert.ToString(r)+":"+my[1]+":"+my[2];
+                    myStrings[i]=z;
+                }
+            }
+            //soting string array for returning time which is the lexicographically smallest one out of all these times.
+            Array.Sort(myStrings ,StringComparer.InvariantCulture);

```

Comment:
what happens when I sort "00:00:01" , "23:59:59"?
Ideally you should use proper datatypes wherever possible i.e. DateTime in this example.